packages/commonwealth/client/scripts/views/pages/new_proposal/cosmos_proposal_form.tsx

```diff
@@ -19,17 +19,18 @@ export const CosmosProposalForm = () => {
   const [cosmosProposalType, setCosmosProposalType] = useState<
     'textProposal' | 'communitySpend'
   >('textProposal');
-  const [deposit, setDeposit] = useState<number | undefined>();
-  const [description, setDescription] = useState();
-  const [payoutAmount, setPayoutAmount] = useState();
-  const [recipient, setRecipient] = useState();
-  const [title, setTitle] = useState();
+  const [description, setDescription] = useState<string>('');
+  const [payoutAmount, setPayoutAmount] = useState<number>(0);
+  const [recipient, setRecipient] = useState<string>('');
+  const [title, setTitle] = useState<string>('');
```

Comment:
that makes sense, do we have an established convention in codebase on this? useful for readability, IMO, happy to change if not necessary

Reply:
I am trying to spread the feedback on many react-related things during the code review. The goal is to have a style guide with best practices for our code base, but this will happen probably when we will be closer to the react switch